### ISSUE TITLE
City not detected

### ISSUE THREAD

**Issue by Trebia** [jeffg2k/SmartCopy#69]
When using SmartCopy in this profile in FamiliySearch: https://familysearch.org/tree/person/KH8W-SJT/details

the city of birth is not recognized properly.

I attach an image with the issue

![smartcopynolocation](https://user-images.githubusercontent.com/9620396/28981494-4fe1bef6-7952-11e7-9215-1045990dc58e.png)

**Comment by jeffg2k** [comment]
Yellow pointer indicates it was ambiguous.  Google does all location translation - this is not something I can fix.

**Comment by jeffg2k** [comment]
I'm going to reopen this... but it's a lower priority as location translation is difficult.  Maybe someone else will pick up the gauntlet.

**Comment by Trebia** [comment]
So... I made some testing. 

- If I replace the city by another (Aldea de San Miguel replacing Camporredondo) the city is properly recognized.
- If I introduce the string "Camporredondo, Valladolid, Castilla y León, Spain" in google maps, it recognizes properly the location. (but I do not know if it is the same API!

Honestly, It does not look to be a problem with translation, as it worked with other city in the same province!

Do you know if there is any way to notify of this issue to google? (I tried hard to look for it!)

**Comment by jeffg2k** [comment]
Hmmm, Here is the string that is being sent.  Seems it should work...
http://maps.googleapis.com/maps/api/geocode/json?language=en&address=Camporredondo,%20Valladolid,%20Castilla%20y%20Le%C3%B3n,%20Spain

Yeah.. this is using a administrative_area_level_4, which is not something I check for.  Will add that to the location test.

**Comment by jeffg2k** [comment]
I'm fixing this for 4.2.8 (fingers crossed it doesn't break other locations).

**Comment by Trebia** [comment]
Great job! Thanks!

Do you want me to perform a couple of testing?

**Comment by jeffg2k** [comment]
I synced it up to GitHub, but I already pushed out 4.2.8 to Google, so you should be able to try it shortly from the main app store update.  To manually update (as oppose to waiting for it automatically), here are some instructions: https://www.howtogeek.com/64525/how-to-manually-force-google-chrome-to-update-extensions/

It's not yet published.. normally takes 30min after I push the button to become available.

**Comment by Trebia** [comment]
So I made some tests, the problem I mentioned is working but, I found another issue.

In this profile: https://familysearch.org/tree/person/MYBF-WPD/details

Location is: "Nuestra Senora De La Asuncion, Camporredondo, Valladolid, Spain"

The location is not an standard location and not recognized by FamilySearch, also the locatino is not recognized by google maps. I found out that actually the place was NOT declared by google maps. So I requested to declare it using the interface of google. I will get the feedback soon and test it.

Keep you updated, if this works it might be interesting to know.
